docs/configuration.adoc

```diff
@@ -20,7 +22,7 @@ But how do we know which tests are important and which ones not?
 There are several strategies that you can choose from which determine what are the *important* tests.
 Currently we have following strategies in place: `new`, `changed`, `affected` and `failed`.
 
-To set them you need to set Java system property `smart.testing` to one or more strategies in comma-separated value form.
+To set them you need to set Java system property `const:core/src/main/java/org/arquillian/smart/testing/Configuration.java[name="SMART_TESTING"]` to one or more strategies in comma-separated value form.
```

Comment:
When I tried to generate it, then the resulting page contains `smart.testing.report.dir` in all cases when `[name="SMART_TESTING"]` is used. It seems that it takes the first variable that starts with the given `"SMART_TESTING"` string. When I moved this variable to the first place inside of the java class, then it is generated correctly.

Reply:
humm probably an error on my regexp :) Thanks for noticing it